biz.aQute.resolve/src/biz/aQute/resolve/BndrunResolveContext.java

```diff
@@ -61,6 +62,7 @@
 	private static final String	BND_AUGMENT					= "bnd.augment";
 	public static final String	RUN_EFFECTIVE_INSTRUCTION	= "-resolve.effective";
 	public static final String	PROP_RESOLVE_PREFERENCES	= "-resolve.preferences";
+	public static final String	NAMESPACE_WHITELIST			= "x-whitelist";
```

Comment:
Definitely do NOT make this field public! It should not be part of the API.

Reply:
right..